docker-compose.yml

```diff
@@ -0,0 +1,167 @@
+version: "2.4"
+
+volumes:
+  keycloak-db: { }
+
+services:
+
+  api:
+    image: ghcr.io/noumenadigital/sbp-dxd/api:latest
+    build:
+      context: api
+    ports:
+      - "8080:8080"
+      - "8000:8000"
+      - "8888:8888"
+    environment:
+      KEYCLOAK_URL: http://keycloak:11000
+      ENGINE_URL: http://engine:12000
+      READ_MODEL_URL: http://read-model:15000
+      HTTP_PORT: 8080
+      HTTP_ADMIN_PORT: 8000
+      IPFS_ENDPOINT: ${IPFS_ENDPOINT:-https://ipfs.infura.io:5001}
+      INFURA_IPFS_PROJECT_ID: ${INFURA_IPFS_PROJECT_ID:-2GaS109tcMuN7KcfAwezGeIJ5rB}
+      INFURA_IPFS_PROJECT_SECRET: ${INFURA_IPFS_PROJECT_SECRET:-998fafd7e3e25d387739441385c71381}
+      PRIVATE_KEY_SBP: MC4CAQAwBQYDK2VwBCIEIFNLBsagBqDpb254CAmo4TnkPi1uCPqICi7pu3xVHzlX
+      PUBLIC_KEY_SBP: MCowBQYDK2VwAyEA2SyvRZj27pEKXuuhJIXSG4l9Qxjt9Krpmw0avvFdHDI=
+      ACCOUNT_HASH_SBP: e5e3607a0925a715429f08d12e82bcf8f996b8113857034fff8a66a067ddd3b3
+      PRIVATE_KEY_USER_1: MC4CAQAwBQYDK2VwBCIEIDUq8MTGOWZ9XHMBeGPFElvwULGUg9aj5maPTv3YKiHd
+      PUBLIC_KEY_USER_1: MCowBQYDK2VwAyEAxxCaPVAtYtgvK4FsuYVi82xtN4+hyd81vWACqJhrGM8=
+      ACCOUNT_HASH_USER_1: d6c48c5c2378e30905bb7916cde5fe5fcae082a0e3eb5093d7435cc7c800bee7
+      PRIVATE_KEY_USER_2: MC4CAQAwBQYDK2VwBCIEIO9We+5WrQPfz3YiVBRPArwOKqH+QuYZRoZ5xdfkMs7p
+      PUBLIC_KEY_USER_2: MCowBQYDK2VwAyEAMPaSs8laS9yWr4u9y3OUqf/l26f+72O9QYUMUPyk8aU=
+      ACCOUNT_HASH_USER_2: 12d91f30f0bc5dbc7d859040f1bd54a9cc2f76fdfa1bee42b0ca92eeed312eb2
+      PRIVATE_KEY_USER_3: MC4CAQAwBQYDK2VwBCIEINHPsohLk/Ijt59+rboz0YB0nm+EYWcnoVDHZxIH+UY7
+      PUBLIC_KEY_USER_3: MCowBQYDK2VwAyEAP7NLRp1Ll7MDyspv20jT/nBmGYrBKY49mvQDjxziY5w=
+      ACCOUNT_HASH_USER_3: 4af7048197bc3f13f8be2d705335f9a291a0905cb0d2505d87962e8ed5aa5dab
```

Comment:
Are these OK??

Reply:
Yes. These are credentials in casper's test environment (https://testnet.cspr.live/), so they're really test accounts with no real value.  In a real system we'd either collect these via the frontend and keep them securely, or we could make users enter their credentials when needed.